test/jarque_bera.jl

```diff
@@ -0,0 +1,58 @@
+using HypothesisTests, Base.Test
```

Comment:
This file will needed to be `include`d in test/runtests.jl. It actually appears that a number of files are missing from that, so we have a bunch of stuff that isn't being tested. 😑